cmd/keymasterd/config.go

```diff
@@ -371,6 +375,14 @@ func loadVerifyConfigFile(configFilename string) (*RuntimeState, error) {
 			return nil, err
 		}
 	}
+	if runtimeState.Config.Okta.Domain != "" {
```

Comment:
//Non-blocking: probably need to make this a TODO to be able to specify priorities for the authenticators

Reply:
Done.